.github/workflows/scripts/script.sh

```diff
@@ -95,6 +97,51 @@ cmd_prefix bash -c "PULP_DATABASES__default__USER=postgres django-admin test --n
 # Run functional tests
 export PYTHONPATH=$REPO_ROOT:$REPO_ROOT/../pulpcore${PYTHONPATH:+:${PYTHONPATH}}
 
+
+if [[ "$TEST" == "upgrade" ]]; then
+  git checkout ci_upgrade_test -- pulp_ansible/tests/
+
+  # Handle app label change:
+  sed -i "/require_pulp_plugins(/d" pulp_ansible/tests/functional/utils.py
+
+  # Running pre upgrade tests:
+  pytest -v -r sx --color=yes --pyargs -capture=no pulp_ansible.tests.upgrade.pre
+
+  # Checking out ci_upgrade_test branch and upgrading plugins
+  cmd_prefix bash -c "cd pulp_ansible; git checkout -f ci_upgrade_test; pip install ."
+  cmd_prefix bash -c "cd pulpcore; git checkout -f ci_upgrade_test; pip install ."
+
+  # Migrating
+  cmd_prefix bash -c "django-admin migrate --no-input"
+
+  # Restarting single container services
+  cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-api"
+  cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-content"
+  cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-resource-manager"
+  cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-worker@1"
+  cmd_prefix bash -c "s6-svc -r /var/run/s6/services/pulpcore-worker@2"
+
+  echo "Restarting in 60 seconds"
+  sleep 60
+
+  # CLI commands to display plugin versions and content data
+  pulp status
+  pulp ansible content list
```

Comment:
@gerrod3 @daviddavis I use pulp-cli here to show we still have content, I thought this was a generic command for all plugins. How can I template the right command here to show we still have data after migration?

Reply:
https://github.com/pulp/plugin_template/blob/master/templates/github/.github/workflows/scripts/script.sh.j2#L147-L153

Reply:
`pulp show --href /pulp/api/v3/content/` There is no generic content list command, we can add it. You could also use `pulp artifact list`, although technically you could have artifacts with no content, but that would be a weird scenario, and if you can have content with no artifacts, which is more likely since on-demand sync doesn't generate any artifacts.